courses/models.py

```diff
@@ -1,50 +1,80 @@
+from django.conf import settings
 from django.db import models
 from embed_video.fields import EmbedVideoField
 
+User = settings.AUTH_USER_MODEL
+
 # Create your models here.
+class Link(models.Model):
+    title = models.CharField(max_length=100)
+    url = models.URLField(max_length=200)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.title
+
+
 class Skill(models.Model):
     name = models.CharField(max_length=100)
+    url = models.ManyToManyField(Link, blank=True)
+    description = models.TextField()
+    image = models.ImageField(upload_to='images/skills/')
+
+    def __str__(self):
+        return self.name
 
 class Instructor(models.Model):
-    name =models.ForeignKey('auth.User',on_delete=models.CASCADE)
+    name =models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
I think think this fiels should be User instead of name, makes more sense i think